### PR TITLE
Pop up version number in ubuntu-r setup script

### PR DESCRIPTION
**Pull request by hetong007**

### CODE REVIEW

**Comment by piiswrong** [comment]
Does it have include the version number?

**Comment by hetong007** [comment]
The R-package has version 0.9.1, while the current setup-utils/install-mxnet-ubuntu-r.sh (https://github.com/dmlc/mxnet/blob/master/setup-utils/install-mxnet-ubuntu-r.sh#L41) looks for version 0.7, thus the script will fail in the last step.

**Comment by piiswrong** [comment]
Can we make `make rpkg` to generate mxnet_current.tar.gz?

**Comment by hetong007** [comment]
It is possible if we add 
```
mv mxnet_*.tar.gz mxnet_current.tar.gz
```
to the end of `make rpkg`

**Comment by piiswrong** [comment]
add the mv command to install-mxnet-ubuntu-r.sh then

File: Makefile [hunk 1]
```diff
@@ -305,6 +305,7 @@ rpkg:
 	Rscript -e "require(mxnet); mxnet:::mxnet.export(\"R-package\")"
 	Rscript -e "require(roxygen2); roxygen2::roxygenise(\"R-package\")"
 	R CMD build --no-build-vignettes R-package
+	mv mxnet_*.tar.gz mxnet_current_r.tar.gz
```
Comment (piiswrong) [inline Makefile]: the docs says:
These commands create the MXNet R package as a tar.gz file that you can install as an R package. To install the R package, run the following command, use your MXNet version number:

    R CMD INSTALL mxnet_0.7.tar.gz

Comment (hetong007) [inline Makefile]: I overlooked these. Incoming fix soon.

Comment (piiswrong) [inline Makefile]: this command doesn't seem to work

Comment (piiswrong) [inline Makefile]: mv *.out c.out
mv: target ‘c.out’ is not a directory

Comment (hetong007) [inline Makefile]: If there's only one wildcard match, this command works. 
I don't think R's native package builder can generate fixed-name tarball.

Comment (piiswrong) [inline Makefile]: How about add that to clean

Comment (thirdwing) [inline Makefile]: I agree to add that into `make clean`

**Pull request merged**